src/types/emr/specimenDefinition/specimenDefinition.ts

```diff
@@ -71,11 +71,10 @@ export const RETENTION_TIME_UNITS = [
 
 export interface TypeTestedSpec {
   is_derived: boolean;
-  specimen_type: Code | null;
   preference: Preference;
   container: ContainerSpec | null;
```

Comment:
```suggestion
  container?: ContainerSpec | null;
```

<img width="515" alt="image" src="https://github.com/user-attachments/assets/f0177441-77a8-430d-b556-245afa29725d" />